code/dynamic_programming/src/longest_increasing_subsequence/longest_increasing_subsequence

```diff
@@ -0,0 +1,21 @@
+def lis(arr)
```

Comment:
Missing file extension. Assuming this is Ruby, use Rubocop to reformat your code.